app/src/main/java/com/github/cmput301f21t44/hellohabits/view/habitevent/CreateEditHabitEventFragment.java

```diff
@@ -70,11 +71,25 @@ public void onViewCreated(@NonNull View view, Bundle savedInstanceState) {
 
         binding.buttonAddHabitEvent.setOnClickListener(v -> submitHabitEvent());
 
+        binding.buttonDeleteHabitEvent.setOnClickListener(v ->
+                new AlertDialog.Builder(requireContext())
+                        .setTitle("Delete Habit Event")
+                        .setMessage("Are you sure you want to delete this habit event?")
+                        .setIcon(R.drawable.ic_launcher_foreground)
+                        .setPositiveButton("YES", (dialog, b) -> deleteHabitEvent())
+                        .setNegativeButton("NO", null).show()
+        );
+
         binding.buttonBack.setOnClickListener(v ->
                 mNavController
                         .navigate(R.id.action_createEditHabitEventFragment_to_viewHabitFragment));
     }
 
+    private void deleteHabitEvent(){
```

Comment:
Make sure to run the auto-format on all the files that you changed so that history doesn't get spammed with "format" commits